flatpak-json2yaml.py

```diff
@@ -89,8 +93,16 @@ def main():
 
         yaml_data = json_to_yaml(h.read())
 
-        with open(out_file, 'wb') as out:
-            out.write(yaml_data)
+        if args.force:
+            flags = O_CREAT|O_WRONLY|O_CLOEXEC
+        else:
+            flags = O_CREAT|O_EXCL|O_WRONLY|O_CLOEXEC
```

Comment:
This change isn't really needed, [Python has had a mode `x` for exclusive creation since 3.3](https://bugs.python.org/issue12760).

Reply:
yeah, I also noticed. And I thought I had pushed that change, but I didn't.
Now i have, though.